src/components/calcite-date-picker-month/calcite-date-picker-month.tsx

```diff
@@ -385,7 +387,7 @@ export class CalciteDateMonth {
           : "hover--outside-range"
       } ${this.isFocusedOnStart() ? "focused--start" : "focused--end"}`
     };
-    return <calcite-date-day {...props} />;
+    return <calcite-date-picker-day {...props} />;
```

Comment:
Nitpick: does this need to be assigned? Seems like it can be inlined, similar to other render methods.

Reply:
Agreed. I'll clean it up.

Reply:
I think some events like `onCalciteDayHover` can be handled in API consistency too.